frontend/packages/dev-console/src/components/pipelines/PipelineRow.tsx

```diff
@@ -28,7 +28,7 @@ const PipelineRow: React.FC<PipelineRowProps> = ({ obj, index, key, style }) =>
   const menuActions = [
     () => startPipeline(PipelineModel, obj, handlePipelineRunSubmit),
     ...(obj.latestRun && obj.latestRun.metadata
-      ? [() => rerunPipeline(PipelineModel, obj, obj.latestRun, handlePipelineRunSubmit)]
```

Comment:
Am I missing something? The story is around adding actions to PipelineRunDetailsPage. Why are we adding/changing PipelineList?